pygeoapi/api.py

```diff
@@ -1419,6 +1421,18 @@ def get_collection_items(
         else:
             skip_geometry = False
 
+        LOGGER.debug('processing cql parameter')
+        cql = request.params.get('cql')
+        if cql is not None:
+            try:
+                cql_ast = parse(cql)
+            except UnexpectedInput as exc:
```

Comment:
`except Exception as err:`

Reply:
Or is there another reason not to import the exceptions into the `api.py` module?

Reply:
We should not import `UnexpectedInput` at the cost of explicitly requiring lark as a dependency.  I think `Exception` is granular enough in this codepath.

Reply:
Fixed by @ximenesuk.